resources/views/sprint/view.blade.php

```diff
@@ -29,16 +29,15 @@
 			</ul>
 
 			@if(Auth::check() && Auth::user()->isInAdminList(env('PHRAGILE_ADMINS')))
-				{!! link_to_route(
-					'delete_sprint_path',
-					'',
-					['sprint' => $sprint->phabricator_id],
-					[
-						'class' => 'btn btn-danger btn-lg glyphicon glyphicon-remove',
-						'title' => 'Delete sprint',
-						'onclick' => 'return confirm("Delete this sprint?")'
-					]
-				) !!}
+				{!! Form::open(['route' => ['delete_sprint_path', $sprint->phabricator_id], 'method' => 'DELETE']) !!}
+					{!! Form::submit(
+						'&#xe014;', [
```

Comment:
Ok, makes sense. Maybe you can add a comment what that encoded character means for glyphicons? That would make it easier to understand what's going on here